examples/TextClassification/README.md

```diff
@@ -1,61 +1,46 @@
 # Deploying Text Classification model
 
-## Installing Deployment plugin
-
-Run the following commands to install deployment plugin
-
-`python setup.py build`
-`python setup.py install`
-
+Download `train.py` and `model.py` from the [respository](https://github.com/pytorch/serve/tree/master/examples/text_classification)
+and subsequently run the following command to train the model in either CPU/GPU.
 
-## Training the model
+CPU: `python train.py AG_NEWS --device cpu --save-model-path  model.pt --dictionary source_vocab.pt`
 
+GPU: `python train.py AG_NEWS --device cuda --save-model-path  model.pt --dictionary source_vocab.pt`
 
-https://github.com/pytorch/serve/tree/master/examples/text_classification
+At the end of the training, model file `model.pt` and vocabulary file `source_vocab.pt` will be stored in the current directory.
 
-Download `train.py` and `model.py` from above mentioned link and run the following command to train the model
+## Installing Deployment plugin
 
-CPU: `python train.py AG_NEWS --device cpu --save-model-path  model.pt --dictionary source_vocab.pt`
+Run the following commands to install deployment plugin
 
-GPU: `python train.py AG_NEWS --device cuda --save-model-path  model.pt --dictionary source_vocab.pt`
+`python setup.py build`
+`python setup.py install`
 
-At the end of the training model file `model.pt` and vocabulary file `source_vocab.pt` will be stored into current directory.
 
-## Starting torchserve
+## Starting TorchServe
 
 create an empty directory `model_store` and run the following command to start torchserve.
 
 `torchserve --start --model-store model_store`
 
+
 ## Creating a new deployment
 
-This example takes text as input and the sample text is placed in `sample_text.txt`
+This example uses the default TorchServe text handler to generate the mar file and inference.
 
 To create a new deployment, run the following command
 
-`python create_deployment.py`
-
-It will create a new deployment named `text_classification`.
-
-Following are the arguments which can be passed to create_deployment script
+``python create_deployment.py --deployment_name text_classification --model_file model.py   --serialized_file model.pt  --extra_files "source_vocab.pt,index_to_name.json"`
```

Comment:
@karthik-77 the command for creating a deployment does not show in code format. Please remove the extra ' at the beginning.

Reply:
made the changes suggested